### DISCUSSION TITLE
Query Managed Property - Date/Time - Include time

### DISCUSSION THREAD

**Discussion by gmanuel7** [microsoft-search/pnp-modern-search#3047] [Q&A]
Using the PnP Modern Search Results web part. We use a managed property "start date" to determine when a specific piece of content should be viewable in the web part. That start date field includes a time and our current query is only looking for Start Date<=Today. This works great to only display results that are set for today or earlier, but if we have content that should not be viewable in the web part until this afternoon, how could we modify the query to include the time.

**Comment by wobba** [comment]
Can you try: `DateProp<={CurrentYear}-{CurrentMonth}-{CurrentDate}T13:00:00Z` or similar? Adjusting the time portion to match what afternoon is in Z time (GMT) for you.

**Comment by gmanuel7** [comment]
And if I wanted the time portion to be dynamic as well to the current time? Could that be achieve with the same inputs you've included for year, month, and date? Content date prop varies and will not always be a set time.

**Reply by wobba** [reply]
Here are the docs for what is supported today https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#date-tokens.

I made a PR (https://github.com/microsoft-search/pnp-modern-search/pull/3067) to support time as well as UTC variants which will be part of the next release (in the dev branch)
